### PR TITLE
Some fixes to develop docker version

### PR DESCRIPTION
**Pull request by einar-lanfranco**
Hi! I did some fixes to the dev docker version.

Please let me know!

Best

### CODE REVIEW

File: .docker/intelmq-full-dev/Dockerfile [hunk 1]
```diff
@@ -18,12 +18,8 @@ RUN sudo chmod +x /opt/entrypoint-dev.sh \
...
-# This section is just for install all the default bots dependencies
-RUN for req in $(find /etc/intelmq/intelmq/bots/ -name "*REQUIREMENTS.txt"); do cat $req >> /tmp/fullrequirements.txt; done
-RUN cat /tmp/fullrequirements.txt | sort | uniq > /tmp/orderfullrequirements.txt
-RUN cat /tmp/orderfullrequirements.txt |grep -Eo '(^[^#]+)' | xargs  -d "\n" -I {} sh -c 'pip3 install "$1"|| exit 0' sh {}
+USER intelmq
 
-USER intelmq:intelmq
-
-ENTRYPOINT [ "/opt/entrypoint-dev.sh" ]
+ENTRYPOINT [ "/opt/entrypoint-dev.sh" ]
```
Comment (sebix) [inline .docker/intelmq-full-dev/Dockerfile]: missing newline at end of file

File: .docker/intelmq-full/Dockerfile [hunk 2]
```diff
@@ -49,7 +48,7 @@ COPY ./intelmq-api /opt/intelmq-api
 
 RUN useradd -d /opt/intelmq -U -s /bin/bash intelmq \
     && adduser intelmq sudo \
-    && echo "%sudo ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers \
+    && echo "intelmq ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers.d/intelmq \
```
Comment (sebix) [inline .docker/intelmq-full/Dockerfile:51]: this creates a security issue. intelmq must not run as root and should not run or required privileges

Comment (einar-lanfranco) [reply .docker/intelmq-full/Dockerfile:51]: I agree, but in this case intelmq is already in sudo group (line 50), in the previous version the entire grupo sudo could execute without password, now its more restricted, only to the intelmq user.
Probably we need to test and understand why it's in sudo.

File: .docker/intelmq-full-dev/Dockerfile (hunk 1 above)
Comment (einar-lanfranco) [reply .docker/intelmq-full-dev/Dockerfile]: I added the new line, i dont know if i need to make a new pull request

Comment (sebix) [reply .docker/intelmq-full-dev/Dockerfile]: When you push commits to the branch, they are all in the PR automatically. Problem solved, thanks :)

File: .docker/intelmq-full/Dockerfile (hunk 2 above)
Comment (sebix) [reply .docker/intelmq-full/Dockerfile:51]: Okay, that's interesting.

File: docker-compose-dev.yml [hunk 3]
```diff
@@ -18,13 +18,14 @@ services:
         networks: 
             - intelmq-internal
     intelmq:
-        build: .docker/intelmq-full-dev
+        # build: .docker/intelmq-full-dev
+        image: certunlp/intelmq3-full-dev:latest
```
Comment (kamil-certat) [inline docker-compose-dev.yml:22]: Could you please set back the original value here? I think the change isn't intentional. I'll merge then

**Pull request merged**